builder/builder-next/builder.go

```diff
@@ -587,7 +587,39 @@ func toBuildkitPruneInfo(opts types.BuildCachePruneOptions) (client.PruneInfo, e
 		// nothing to do
 	case 1:
 		var err error
-		until, err = time.ParseDuration(untilValues[0])
```

Comment:
Instead of implementing the parsing here, I think it would be better to use the same implementation as is used by the other endpoints (see the original implementation in https://github.com/moby/moby/pull/29226); https://github.com/moby/moby/blob/48cfe3f0872647bf0f52e6ba65022e1859e808e8/api/types/time/timestamp.go


(TBH, in hindsight, we should've done the parsing itself in the client, and always send a fixed format timestamp over the API, but I guess that's too late to change)

Reply:
Good catch! I've updated it to use the method you've suggested.